Formula/openshift-cli.rb

```diff
@@ -16,6 +16,8 @@ class OpenshiftCli < Formula
   end
 
   devel do
+    depends_on "socat"
```

Comment:
@DomT4 addressed in https://github.com/Homebrew/homebrew-core/pull/4105/commits/cbb1b2c06035bf05ceea1f4a422505cd13a206ad